### PR TITLE
added videos, fixed a link, updated image

### PR DESCRIPTION
**Pull request by YashKumarVerma**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Images are `240 x 240` [w x h].
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request
- Added 5 videos to the GCI Site
- Fixed link of a video, now starts at 0 seconds, 0 offset. 

Demo:
https://yashkumarverma.github.io/gci17.fossasia.org/

image

### CODE REVIEW

**Review by anshumanv — APPROVED** [review]
LGTM

File: _data/video.yml [hunk 1]
```diff
@@ -19,9 +19,29 @@
 
 - url: https://www.youtube.com/watch?v=XvoHY3QCJtw
   title: Labyrinth
-  img: labyrinth.jpg
+  img: verma-labyrinth.png
   author_name: Yash Kumar Verma
   
+- url: https://www.youtube.com/watch?v=yc_d5JJ--vQ
+  title: SUSI AI on Kubernetes
+  img: verma-susu-kubernetes.png
```
Comment (anshumanv) [inline _data/video.yml]: Everything seems to be working but I think you can use a better file name here. I mean why `susu`?

File: _data/video.yml [hunk 2]
```diff
@@ -19,9 +19,29 @@
...
+  author_name: Yash Kumar Verma
+
+- url: https://www.youtube.com/watch?v=eKRFpH3P0nw
+  title: Compiling Wine
+  img: verma-wine.png
+  author_name: Yash Kumar Verma
+
+- url: https://www.youtube.com/watch?v=5se37I6BGy0
+  title: Setting Up SUSI Server
+  img: verma-susi-servver.png
```
Comment (anshumanv) [inline _data/video.yml]: Why `servver`? Things will work but it doesn't look good.

**Review by anshumanv — COMMENTED** [review]
See comments please.

**Review by rupav — CHANGES_REQUESTED** [review]
Just change to more appropriate names of the images as suggested by @anshumanv . Rest LGTM. And images are looking fantastic!

**Review by akshatnitd — COMMENTED** [review]
@YashKumarVerma Please make the suggested changes, so that we can merge!

File: _data/video.yml (hunk 1 above)
Comment (YashKumarVerma) [reply _data/video.yml]: typo :laugh:

Comment (anshumanv) [reply _data/video.yml]: Haha, I knew. :laughing:

Comment (anshumanv) [reply _data/video.yml]: Btw images are well made, :+1:

**Review by isuruAb — COMMENTED** [review]
please do requested changes. rest looks good

**Comment by YashKumarVerma** [comment]
@rupav @anshumanv @isuruAb 

Done :)

**Review by rupav — APPROVED** [review]

**Pull request merged**